src/main/java/org/dependencytrack/resources/v1/FindingResource.java

```diff
@@ -177,5 +181,115 @@ public Response analyzeProject(
         }
     }
 
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(
+            value = "Returns a list of all findings",
+            response = Finding.class,
+            responseContainer = "List",
+            responseHeaders = @ResponseHeader(name = TOTAL_COUNT_HEADER, response = Long.class, description = "The total number of findings")
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 401, message = "Unauthorized"),
+    })
+    @PermissionRequired(Permissions.Constants.VIEW_VULNERABILITY)
+    public Response getAllFindings(@ApiParam(value = "Show inactive projects")
+                                   @QueryParam("showInactive") boolean showInactive,
+                                   @ApiParam(value = "Show suppressed findings")
+                                   @QueryParam("showSuppressed") boolean showSuppressed,
+                                   @ApiParam(value = "Filter by severity")
+                                   @QueryParam("severity") String severity,
```

Comment:
I would prefer to not omit it, since it's an important filter that I will need. When I first testet this feature we did a small calculation how much we would miss, and it was a very small percentage that is affected by #2474. Maybe a small hint in the UI, that there are exceptional cases due to technical reasons where this filter will provide incorrect results until a future version could solve this?